hls4ml/model/hls_model.py

```diff
@@ -1,4 +1,5 @@
 from __future__ import print_function
+from email.policy import default
```

Comment:
What is this needed for?

Reply:
Nothing, leftover from testing I believe.